indico/modules/events/registration/models/forms.py

```diff
@@ -32,10 +32,11 @@
 
 
 class ModificationMode(RichIntEnum):
-    __titles__ = [None, L_('Until modification deadline'), L_('Until payment'), L_('Never')]
+    __titles__ = [None, L_('Until modification deadline'), L_('Until payment'), L_('Never'), L_('Until approved')]
```

Comment:
hm, when discussing about this with @OmeGak recently i suggested that "While pending approval" may be a good title that's clearer. no strong opinion on this though

Reply:
Done

Reply:
here it is @pferreir